frontend/src/app/patients/AddPatient.test.tsx

```diff
@@ -31,7 +32,7 @@ jest.mock("../utils/smartyStreets", () => ({
   suggestionIsCloseEnough: () => false,
 }));
 
-const fillOutForm = (
+export const fillOutForm = (
```

Comment:
hmm why the export?

Reply:
Keen eye - I was going to use this elsewhere at one point but backtracked. Removed in d0801f3.